personal-practice/arrays.java

```diff
@@ -0,0 +1,28 @@
+import java.awt.*;
+import java.util.Scanner;
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+public class arrays {
```

Comment:
User [UpperCamelCase](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names) for class names and the containing java file.